### ISSUE TITLE
Minetest resource manager

### ISSUE THREAD

**Issue by paper-pauper** [minetest/minetest#3718]
I think it would be great if Minetest shipped with a resource manager to help users find and download subgames, mods and texture packs, as long as no files released under nonfree licenses are distributed (to encourage free licensing as well as prevent Minetest from being banned from distros like Trisquel and GuixSD).

A central database for subgames, mods and texture packs is needed anyway: people can't simply hunt for them in the forums.

**Comment by LeMagnesium** [comment]
Well we used to have [this](https://forum.minetest.net/viewtopic.php?id=6883) but it is now down (see [here](https://forum.minetest.net/mmdb/)). It was combined with a mod store in the game's menu, very similar to what you wrote about : 

![](https://lut.im/8metX3C4Ao/6vhRCPs7kc6TWe6V)

![](https://lut.im/7KaCQtJ3Bb/9NHch34kT3jWUqMZ)

For some reason all of this was removed.  So if it existed at some point in the past and fell apart, although I don't really know why, I am not sure it will ever be created again. However, if it were well implemented and working correctly it would be a great feature, but it means also people would have to dedicate time to maintaining those repositories/lists of mods and subgames.
